src/FSharp Examples/AlexNet.fs

```diff
@@ -0,0 +1,194 @@
+module TorchSharp.Examples.AlexNet
+
+open System
+open System.IO
+open System.Diagnostics
+
+open TorchSharp
+open TorchSharp.Tensor
+open TorchSharp.NN
+
+open type TorchSharp.NN.Modules
+open type TorchSharp.TorchScalar
+
+/// <summary>
```

Comment:
I think remove the XML docs here and elsewhere. Just use `//` docs -  or else put the XML docs on the outer module for the whol file (the `module` currently on the first line.  

As you have it the XML docs are attaching to the first 'let' that follows the docs, which is `trainBatchSize` in this case

Reply:
Will do.